src/shared-components/dialogModal/index.tsx

```diff
@@ -42,10 +48,13 @@ const getDomNode = () =>
  */
 export const DialogModal = ({
   children,
+  modalColor,
   onCloseIconClick,
   title = '',
   ...rest
 }: DialogModalProps) => {
+  const theme: ThemeType = useTheme();
```

Comment:
We've got built-in typing with `useTheme` so we can remove the annotation:

<img width="229" alt="Screen Shot 2021-02-11 at 12 14 51 PM" src="https://user-images.githubusercontent.com/13544620/107679920-c86b4480-6c62-11eb-8a84-6329f932520a.png">